# HG changeset patch
# User Rik <rik@octave.org>
# Date 1351389231 25200
#      Sat Oct 27 18:53:51 2012 -0700
# Node ID de751531e548edf58de7660f9826e23d8449d097
# Parent  ed6385e234208426909ea67fa5e78179d62e73e2
test: Add demos for bar.m and barh.m

* bar.m, barh.m: Add %!demos.

diff --git a/scripts/plot/bar.m b/scripts/plot/bar.m
--- a/scripts/plot/bar.m
+++ b/scripts/plot/bar.m
@@ -90,10 +90,20 @@
 ## Author: jwe
 
 function varargout = bar (varargin)
   varargout = cell (nargout, 1);
   [varargout{:}] = __bar__ (true, "bar", varargin{:});
 endfunction
 
 
-%% FIXME: Need demo or test for function
+%!demo
+%! clf;
+%! y = rand (10, 1);
+%! bar (y);
 
+%!demo
+%! clf;
+%! h = bar (rand (5, 3));
+%! set (h(1), 'facecolor', 'r')
+%! set (h(2), 'facecolor', 'g')
+%! set (h(3), 'facecolor', 'b')
+
diff --git a/scripts/plot/barh.m b/scripts/plot/barh.m
--- a/scripts/plot/barh.m
+++ b/scripts/plot/barh.m
@@ -49,10 +49,20 @@
 ## Author: jwe
 
 function varargout = barh (varargin)
   varargout = cell (nargout, 1);
   [varargout{:}] = __bar__ (false, "barh", varargin{:});
 endfunction
 
 
-%% FIXME: Need demo or test for function
+%!demo
+%! clf;
+%! x = rand (10, 1);
+%! barh (x);
 
+%!demo
+%! clf;
+%! h = barh (rand (5, 3));
+%! set (h(1), 'facecolor', 'r')
+%! set (h(2), 'facecolor', 'g')
+%! set (h(3), 'facecolor', 'b')
+
